include/SugarFolders/SugarFolders.php

```diff
@@ -613,13 +625,13 @@ public function getCountUnread($folderId)
 
 
     /**
-     * Convenience method, pass a SugarBean and User to this to add anything to a given folder
+     * Convenience method, pass a SugarBean
      *
      * @param string $bean
-     * @param User   $user
+     *
      * @return boolean
      */
-    public function addBean($bean, $user = null)
+    public function addBean($bean)
```

Comment:
I don't think the $bean should be a string. Use SugarBean type hinting and corrigate the phpdoc above please.